tests/particle_attributes/Makefile

```diff
@@ -0,0 +1,15 @@
+input: default
+default: clean
+	gmsh -2 -bin square.geo -o square.msh
+clean: 
+	rm -rf *.msh *.halo *.vtu *.pvtu *.log* *.err* *.stat blob_[012] *.particles.* *flredecomp* particle_attributes_? *~
+
+logs: 
```

Comment:
`logs`, `grep` and `debug` seem very specific and nothing to do with the actual test